gslib/tests/test_copy_helper_funcs.py

```diff
@@ -413,6 +413,21 @@ def testSetContentTypeFromFile(self):
       _SetContentTypeFromFile(src_url_stub, dst_obj_metadata_mock)
     self.assertEqual('text/plain', dst_obj_metadata_mock.contentType)
 
+  def testSetsContentTypesForCommonFileExtensionsCorrectly(self):
+    extension_rules = copy_helper.COMMON_EXTENSION_RULES.items()
+    for extension, expected_content_type in extension_rules:
+      dst_obj_metadata_mock = mock.MagicMock(contentType=None)
+      src_url_stub = mock.MagicMock(object_name='file' + extension,
+                                    **{
```

Comment:
This is neat, but isn't this simpler to understand syntax?
```
src_url_stub.IsFileUrl.return_value = True
src_url_stub.IsStream...
```

Reply:
Sure!